pkg/config/config.go

```diff
@@ -94,6 +94,10 @@ const (
 
 	// maxExternalMetricsProviderChunkSize ensures batch queries are limited in size.
 	maxExternalMetricsProviderChunkSize = 35
+
+	// DefaultLocalProcessCollectorInterval is the interval at which processes are collected and sent to the workloadmeta
+	// in the core agent if the process check is disabled.
+	DefaultLocalProcessCollectorInterval = 1 * time.Minute
```

Comment:
How does this parameter influence performance? We might miss short-living processes here.

Reply:
We actually always miss short-lived processes even in the process check. 😅 

Live processes isn't actually meant to collect short-lived processes, which is why we have another check for process events.

The goal here is to match the interval of the cluster agent check, so let me know if this 1 minute interval doesn't sound accurate